scaffolds/streamlit_on_cloudrun/image_classification/README.md

```diff
@@ -0,0 +1,16 @@
+# Streamlit Application - Image Classification
+
+This directory includes files to build and deploy an image classification application on streamlit.
+
+For all the step by step instructions, refer to the `train_and_deploy.ipynb` which includes detailed explanations on each file.
+
+Or, if you want to quickly deploy and check the application, follow the steps below:
+
+1. Open `deploy.sh` and change the PROJECT_ID variable to your project id.
```

Comment:
This step fails if you don't run the notebook first because the `export` folder is missing if you don't run the notebook first.

Reply:
`mkdir export` command is included in the `deploy.sh`. Maybe failed with another problem?